packages/utils/src/lib/reports/utils.ts

```diff
@@ -30,27 +30,35 @@ export function formatScoreWithColor(
   const styledNumber = options?.skipBold
     ? formatReportScore(score)
     : style(formatReportScore(score));
-  return `${getRoundScoreMarker(score)} ${styledNumber}`;
+  return `${scoreMarker(score)} ${styledNumber}`;
 }
 
-export function getRoundScoreMarker(score: number): string {
-  if (score >= SCORE_COLOR_RANGE.GREEN_MIN) {
-    return '🟢';
-  }
-  if (score >= SCORE_COLOR_RANGE.YELLOW_MIN) {
-    return '🟡';
-  }
-  return '🔴';
-}
+export type MarkerTypes = 'circle' | 'square';
```

Comment:
- union type should be singular (notice how you end up using `markerType: MarkerTypes` otherwise)
- "shape" would be more specific than "type"

```suggestion
export type MarkerShape = 'circle' | 'square';
```